apps/service_providers/llm_service/history_managers.py

```diff
@@ -89,6 +89,7 @@ def __init__(
         self.max_token_limit = max_token_limit
         self.chat_model = chat_model
         self.trace_service = trace_service
+        self.ai_message = ""
```

Comment:
We set this attribute [here](https://github.com/dimagi/open-chat-studio/blob/02fdb02c8c669426f197165fc7236357be595e28/apps/service_providers/llm_service/history_managers.py#L62), but it was never initialized to a default value. Now it is